### DISCUSSION TITLE
DVTPortal: Error: Code=1100 "Your session has expired. Please log in." from Jenkins

### DISCUSSION THREAD

**Discussion by garyhooper** [fastlane/fastlane#21387] [Q&A]
I am seeing this error whenever I execute fastlane from Jenkins:
```
DVTPortal: Service '<DVTPortalViewDeveloperService: 0x60000058ce40; action='viewDeveloper'>' encountered an unexpected result code from the portal ('1100')
DVTPortal: Error: Error Domain=DVTPortalServiceErrorDomain Code=1100 "Your session has expired. Please log in." UserInfo={...}
```
If I instead login as the Jenkins user, and execute the fastlane build from the command line, all works successfully as expected.

Prior to executing in Jenkins, I unlock my keychain, and request that it remain open.

I've searched high and low with no solution.  Can anyone provide ideas as to how to resolve this?

I'm on Fastlane 2.213.0; MacOS 13.4; Xcode 14.3.1 & Xcode 15.0 B3.  I use `fastlane match`, and an API key to communicate with the App Store.

**Comment by sauravexodus** [comment]
Same here

**Comment by freon27** [comment]
I'm seeing the same error during the build step after using cert / sigh.

Locally using the same API key it still works
